packages/codept/codept.0.11.0/opam

```diff
@@ -0,0 +1,35 @@
+opam-version: "2.0"
+maintainer: "Florian Angeletti <octa@polychoron.fr>"
+homepage: "https://github.com/Octachron/codept"
+bug-reports: "https://github.com/Octachron/codept/issues"
+license: "gpl-3"
+dev-repo: "git+https://github.com/Octachron/codept.git"
+build: [
+  ["dune" "build" "-p" name]
+]
+run-test: [
+  ["dune" "runtest"]
+]
+depends: ["dune" "menhir" {build} "ocaml" {>="4.03" & < "4.14~"}]
```

Comment:
```suggestion
depends: [
  "ocaml" {>="4.03" & < "4.14~"}
  "dune" {>= "2.7"}
  "menhir" {build}
  "ocamlfind" {with-test}
]
```
If I looked correctly, the dune-project file requires dune 2.7 and the tests call ocamlfind

Reply:
I am going to relax the dune requirement to 2.5 . Similarly, I will update the test to call `ocamlc -where` rather than ocamlfind to locate `compiler-libs`.